tests/functional/conftest.py

```diff
@@ -187,6 +187,17 @@ def get_docker_image_os_version_from_container():
     return os_distname
 
 
+def get_ov_and_ovms_version_from_container():
```

Comment:
Shouldn't this be exluded from this MR?

Reply:
This is cherry-picked from develop to unblock CI functional tests. There should be no issue when merging feature branch with develop later on.